src/components/ComputeModal.js

```diff
@@ -941,6 +942,9 @@ export const ComputeModalBase = ({ onDismiss, onSuccess, runtimes, persistentDis
           h(IdContainer, [
             id => div({ style: { gridColumnEnd: 'span 4', marginTop: '0.5rem' } }, [
               label({ htmlFor: id, style: computeStyles.label }, ['Compute type']),
+              h(InfoBox, { style: { marginLeft: '0.5rem', display: requiresStandard() || requiresSpark ? '' : 'none' } }, [
+                'Only the compute types compatible with the selected application configuration are made available below.'
```

Comment:
maybe just check quickly with our UX partner, Tiffany Chan, about the wording here if you haven't :)